react-forms/src/DatePicker.tsx

```diff
@@ -182,9 +184,19 @@ export default function DateTimePicker<T>(props: IProps<T>) {
 
             <input
                 data-help={guid}
-                className={"gpa-gemstone-datetime form-control" + (props.Valid(props.Field) ? '' : ' is-invalid')}
+                className={`gpa-gemstone-datetime form-control ${!props.Valid(props.Field) || feedbackMessage ? 'is-invalid' : ''}`}
                 type={props.Type === undefined ? 'date' : props.Type}
                 onChange={(evt) => {
+                    const inputVal = evt.target.value ?? "";
+                    const date = moment(inputVal, boxFormat, true);
+                    const validStartDate = moment("1753-01-01", "YYYY-MM-DD");
+
+                    if (date.isValid() && date.isSameOrAfter(validStartDate)) {
```

Comment:
That is ok but in terms of checking but if the internal validation fails (e.g. not a valid date or before 1753) you can not set the record (`props.Setter`), otherwhise you will not trigger external validity checks. You'll need to add this check somewhere around Line 84 as well to avoid that.